content/200-concepts/200-database-connectors/03-postgresql.mdx

```diff
@@ -147,63 +148,64 @@ The PostgreSQL connector maps the [scalar types](/concepts/components/prisma-sch
-| `line`                                      | `Unsupported` |  Not yet  |                                          |                                                                                                                                                                                    |
-| `lseg`                                      | `Unsupported` |  Not yet  |                                          |                                                                                                                                                                                    |
-| `box`                                       | `Unsupported` |  Not yet  |                                          |                                                                                                                                                                                    |
-| `path`                                      | `Unsupported` |  Not yet  |                                          |                                                                                                                                                                                    |
-| `polygon`                                   | `Unsupported` |  Not yet  |                                          |                                                                                                                                                                                    |
-| `circle`                                    | `Unsupported` |  Not yet  |                                          |                                                                                                                                                                                    |
-| Composite types                             | n/a           |  Not yet  |                                          |                                                                                                                                                                                    |
-| Domain types                                | n/a           |  Not yet  |                                          |                                                                                                                                                                                    |
+### Mapping between PostgreSQL database column types to Prisma scalar and native types
+
+When [introspecting](/concepts/components/introspection) a PostgreSQL database, the database types are mapped to Prisma types according to the following table.
+When [creating a migration](/concepts/components/prisma-migrate) or [prototyping your schema](/concepts/components/prisma-migrate/db-push) the table is also used - in the other direction.
```

Comment:
This needs a complete rewrite to not be super awkward - but I think _this_ is the information we want to convey here.

Reply:
```suggestion
- When [introspecting](/concepts/components/introspection) a PostgreSQL database, the database types are mapped to Prisma types according to the following table.
- When [creating a migration](/concepts/components/prisma-migrate) or [prototyping your schema](/concepts/components/prisma-migrate/db-push) the table is also used - in the other direction.
```